.github/workflows/release.yml

```diff
@@ -1,9 +1,16 @@
 name: Build distribution
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+      - main
+    tags:
+      - "20*" # Push events to matching 20*, i.e. 2022.2.0, 2099.8.1
 
 jobs:
-  test:
+  release:
+    needs: [ test ]
```

Comment:
I'd rather leave this up to human decision. We shouldn't create a tag unless things are working as expected. 

I could imagine a situation where we want to release despite there being a known test issue.